src/codecs.ts

```diff
@@ -728,6 +729,8 @@ export const ConsentManageExperienceInput = t.intersection([
         country: valuesOf(IsoCountryCode),
       }),
     ),
+    /** How to handle consent expiry */
+    onConsentExpiry: valuesOf(OnConsentExpiry),
```

Comment:
this field needs to be used in conjunction with `consentExpiry` as well which is the duration for when to expire the consent

Reply:
ill add that

Reply:
done